stable/artifactory/README.md

```diff
@@ -186,6 +186,38 @@ helm install --name artifactory --set artifactory.license.secret=artifactory-lic
 **NOTE:** This method is relevant for initial deployment only! Once Artifactory is deployed, you should not keep passing these parameters as the license is already persisted into Artifactory's storage (they will be ignored).
 Updating the license should be done via Artifactory UI or REST API.
 
+### Configure NetworkPolicy
+
+NetworkPolicy specifies what ingress and egress is allowed in this namespace. It is encouraged to be more specific whenever possible to increase security of the system.
+
+In the `networkpolicy` section of values.yaml you can specify a list of NetworkPolicy objects.
+
+For podSelector, ingress and egress, if nothing is provided then a default `- {}` is applied which is to allow everything.
+
+A full (but very wide open) example that results in 2 NetworkPolicy objects being created:
+```
+networkpolicy:
+  # Allows all ingress and egress to/from artifactory primary and member pods.
```

Comment:
`artifactory` only. No `primary` and `member`.

Reply:
Done.